src/components/dialogs/two-windings-transformer/phase-tap-changer-pane.js

```diff
@@ -0,0 +1,563 @@
+                              ),
+                        alpha: isNaN(
+                            parseFloat(
+                                val[
+                                    intl.formatMessage({
+                                        id: 'ImportFileAlpha',
+                                    })
+                                ]
+                            )
+                        )
+                            ? 1
+                            : parseFloat(
+                                  val[
+                                      intl.formatMessage({
+                                          id: 'ImportFileAlpha',
+                                      })
+                                  ]
+                              ),
+                        isEdited: true,
+                    };
+                });
+                let tapValues = Object.values(
+                    rows.map((row) => {
+                        return parseInt(row.tap, 10);
+                    })
+                );
+                let tempLowTapPosition = Math.min(...tapValues);
+                let tempHighTapPosition = Math.max(...tapValues);
+
+                let tempFormValues = {
+                    ...formValues,
+                    phaseTapChanger: {
+                        ...formValues.phaseTapChanger,
+                        lowTapPosition: tempLowTapPosition,
+                        highTapPosition: tempHighTapPosition,
+                    },
+                };
+                setFormValues(tempFormValues);
+                handlePhaseTapRows(rows);
+            },
+        });
+    };
+
+    const handleCreateAlphaTapRule = (lowTapAlpha, highTapAlpha) => {
+        let tempRows = phaseTapRows;
+
+        if (tempRows.length > 1) {
+            let alphaInterval =
+                (highTapAlpha - lowTapAlpha) / (tempRows.length - 1);
+            let currentAlpha = lowTapAlpha;
+
+            tempRows.forEach((row, index) => {
+                tempRows[index].alpha = currentAlpha;
+                currentAlpha += alphaInterval;
+            });
+            handlePhaseTapRows(tempRows);
+        }
+    };
+
+    return (
+        <>
+            <Grid container spacing={2}>
+                <Grid item container spacing={2}>
+                    <Grid item xs={4}>
+                        {phaseTapChangerEnabledField}
+                    </Grid>
+                </Grid>
+                <Grid item xs={4}>
+                    {gridItem(regulationModeField, 12)}
+                </Grid>
+
+                <Grid item container spacing={2}>
+                    <Grid item xs={4}>
+                        {regulatingField}
+                    </Grid>
+
+                    <Grid item xs={4}>
+                        {regulationMode !== REGULATION_MODES[2].id &&
+                            currentLimiterRegulatingValueField}
+                        {regulationMode === REGULATION_MODES[2].id &&
```

Comment:
See my other comment in constants.js.
I think it's better to call this with `REGULATION_MODES.ACTIVE_POWER_CONTROL.id`

Reply:
✓